docs/index.rst

```diff
@@ -83,6 +83,21 @@ On this tab a number of configuration parameters are available;
     | |opcua_2| |
     +-----------+
 
+The Filtering tab allows setting the filtering config to include/exclude the OPCUA nodes to subscribe to.
+
++-------------------+
+| |opcua_filtering| |
++-------------------+
+
+  - **Regular expression**: The regular expression (regex) to be matched against the Browse Name of the node. The regex has to match the browse name completely.
```

Comment:
The regex has to match the browse name completely -> The regex has to match the Browse Name exactly